### PR TITLE
fixes bug 1005224 - log failed correlations reports

### PR DESCRIPTION
**Pull request by peterbe**
Cosmetic small improvement.

### CODE REVIEW

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js [hunk 1]
```diff
@@ -14,6 +14,12 @@ var SignatureCorrelations = (function() {
                     if (response.count) {
                         $('li.correlations').show();
                     }
+                    if (response.errors) {
```
Comment (adngdb) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js]: An empty array evaluates to true in JavaScript (try ``if ([]) console.log('yes');`` in your console), so this test should be different. Maybe ``if (response.errors.length) {`` instead?

**Comment by adngdb** [comment]
I'm not sure there is much value in doing this, but it's still better than doing nothing. :)

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js (hunk 1 above)
Comment (peterbe) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js]: The server doesn't include the key `errors` if there are no errors. So the code works as is. Arguably, perhaps it should. And then the if statement would need to be smarter.

Comment (adngdb) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js]: It actually does. You changed that behavior in this PR. :)

Comment (peterbe) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js]: Good catch! Pushed a fix for this now.

**Pull request merged**